client/app/services/dashboard.js

```diff
@@ -154,6 +154,24 @@ function transformResponse(data) {
   return data;
 }
 
+function transformRecentResponse(data) {
```

Comment:
Let's not mix up the transform and "recent" functions and have two separate `.then`:

```js
{
  recentDashboards: params =>
    axios
      .get("api/dashboards", { params })
      .then(transformResponse)
      .then(doSomeMappingHere),
}
```
(need to check in what order it makes sense)

Reply:
For sure, i also think this is a better aproach, i'll apply this changes asap.

Reply:
@gabrieldutra Done!